multiqc/modules/longranger/longranger.py

```diff
@@ -0,0 +1,330 @@
+                'suffix': 'X'
+        }
+        self.headers['zero_coverage'] = {
+                'scale': 'PuBu-rev',
+                'title': 'Zero cov',
+                'modify': lambda x: float(x) * 100.0,
+                'suffix': '%',
+                'description': 'Fraction of non-N bases in the genome with zero coverage.'
+        }
+        self.headers['mapped_reads'] = {
+                'scale': 'PuBu',
+                'title': 'Mapped',
+                'modify': lambda x: float(x) * 100.0,
+                'suffix': '%',
+                'description': 'Number of input reads that were mapped.'
+        }
+        self.headers['pcr_duplication'] = {
+                'scale': 'PuBu-rev',
+                'title': 'PCR Dup',
+                'description': 'Fraction of reads marked as PCR duplicates. To be marked as PCR duplicates, reads must have the same mapping extents on the genome and the same 10x barcode.',
+                'suffix': '%',
+                'modify': lambda x: float(x) * 100.0
+        }
+        self.headers['on_target_bases'] = {
+                'scale': 'PuBu',
+                'title': 'On target',
+                'description': 'Fraction of aligned bases mapped with the target regions in targeted mode. Only bases inside the intervals of target BED file are counted.',
+                'suffix': '%',
+                'modify': lambda x: 0 if x=="" else float(x) * 100.0
+        }
+        self.headers['r1_q20_bases_fract'] = {
+                'scale': 'PuBu',
+                'title': 'R1 Q20',
+                'description': 'Fraction of bases in R1 with base quality >= 20.',
+                'hidden': True,
+                'suffix': '%',
+                'modify': lambda x: float(x) * 100.0
+        }
+        self.headers['r1_q30_bases_fracts'] = {
+                'scale': 'PuBu',
+                'title': 'R1 Q30',
+                'description': 'Fraction of bases in R1 with base quality >= 30.',
+                'hidden': True,
+                'suffix': '%',
+                'modify': lambda x: float(x) * 100.0
+        }
+        self.headers['r2_q20_bases_fract'] = {
+                'scale': 'PuBu',
+                'title': 'R2 Q20',
+                'description': 'Fraction of bases in R2 with base quality >= 20.',
+                'hidden': True,
+                'suffix': '%',
+                'modify': lambda x: float(x) * 100.0
+        }
+        self.headers['r2_q30_bases_fract'] = {
+                'scale': 'PuBu',
+                'title': 'R2 Q30',
+                'description': 'Fraction of bases in R2 with base quality >= 30.',
+                'hidden': True,
+                'suffix': '%',
+                'modify': lambda x: float(x) * 100.0
+        }
+        self.headers['si_q20_bases_fract'] = {
```

Comment:
This is a mega huge table, though you've hidden most columns. Normally I'd say that you should just remove these columns, or split into several tables / plots etc. But I have a feeling that you're going to say that it's useful to have them there so that you can do scatter plots..?

But still, may be worth removing some of these :)

Reply:
Yeah. These R1, R2, si, etc, they're a bit much.